### PR TITLE
change variable from JButton to JTextfield

### PR DESCRIPTION
**Pull request by 9728627**
The Copyright Button #76
It should either do something or not be a button.
50 XP

I have changed it to a text instead of a button and cleaned the garbages.

### CODE REVIEW

**Comment by 9728627** [comment]
It says all checks have failed. However, I did not change anything else besides the copyright button. Will you merge it to the master whenever I posted it or after the lab is due?

File: src/edu/ucsb/cs56/projects/games/fish_animation/Menu.java [hunk 1]
```diff
@@ -31,7 +31,8 @@
 
 class Menu implements ActionListener {
 
-	private JButton Play, Instruction, Exit, Resume, Easy, Medium, Hard, Back, Menu, Credit, PlayBGM, PauseBGM, HighScore, NextPage;
+	private JButton Play, Instruction, Exit, Resume, Easy, Medium, Hard, Back, Menu, PlayBGM, PauseBGM, HighScore, NextPage;
+    private JTextField Credit;
```
Comment (xavierholt) [inline src/edu/ucsb/cs56/projects/games/fish_animation/Menu.java]: Fix your indentation.

File: src/edu/ucsb/cs56/projects/games/fish_animation/Menu.java [hunk 2]
```diff
@@ -108,7 +109,7 @@ public void makegui() {
 		Instruction.addActionListener(this);
 		HighScore.addActionListener(this);
 		Exit.addActionListener(this);
-		Credit.addActionListener(this);
+		//Credit.addActionListener(this);
```
Comment (xavierholt) [inline src/edu/ucsb/cs56/projects/games/fish_animation/Menu.java]: Clean out your unused code - don't just comment it.

File: src/edu/ucsb/cs56/projects/games/fish_animation/Menu.java [hunk 3]
```diff
@@ -190,25 +191,12 @@ public void mouseExited(java.awt.event.MouseEvent evt) {
 		});
 
 		Credit.setOpaque(false);
-		Credit.setContentAreaFilled(false);
 		Credit.setBorder(BorderFactory.createLineBorder(Color.white));
-		Credit.setForeground(Color.white);
+        Credit.setBackground(new Color(0,0,0,0));
```
Comment (xavierholt) [inline src/edu/ucsb/cs56/projects/games/fish_animation/Menu.java]: Another indentation fix!

File: src/edu/ucsb/cs56/projects/games/fish_animation/Menu.java [hunk 4]
```diff
@@ -190,25 +191,12 @@ public void mouseExited(java.awt.event.MouseEvent evt) {
 		});
 
 		Credit.setOpaque(false);
-		Credit.setContentAreaFilled(false);
 		Credit.setBorder(BorderFactory.createLineBorder(Color.white));
```
Comment (xavierholt) [inline src/edu/ucsb/cs56/projects/games/fish_animation/Menu.java]: Maybe get rid of the border, too?  It still _looks_ like a button...

**Comment by xavierholt** [comment]
Any update on this one?  I'm not really able to look at the PRs that include this one until it gets merged - it's too hard to tell what's going on.

If you want to make fixes to this branch, you should be able to `git checkout copyrightButton` and work from there.

**Comment by 9728627** [comment]
ok, fixed. Sorry about the late reply.

**Pull request merged**